Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -305,17 +305,17 @@ IDD_TAB_CHAT DIALOGEX 0, 0, 176, 142
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_CHILD | WS_VISIBLE
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    LTEXT           "History",IDC_STATIC_CHANMESSAGES,0,0,0,0
+//    LTEXT           "History",IDC_STATIC_CHANMESSAGES,0,0,0,0
     CONTROL         "",IDC_RICHEDIT_CHANMESSAGES,"RICHEDIT20W",WS_BORDER | WS_VSCROLL | WS_TABSTOP | 0x1844,0,0,176,125
-    LTEXT           "Message",IDC_STATIC_CHANMESSAGE,0,0,0,0
+//    LTEXT           "Message",IDC_STATIC_CHANMESSAGE,0,0,0,0
     EDITTEXT        IDC_EDIT_CHANMESSAGE,0,126,176,14,ES_AUTOHSCROLL
 END
 
 IDD_TAB_FILES DIALOGEX 0, 0, 186, 138
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_CHILD | WS_VISIBLE
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    LTEXT           "Files list",IDC_STATIC_FILESLIST,0,0,0,0
+//    LTEXT           "Files list",IDC_STATIC_FILESLIST,0,0,0,0

```

Comment:
Can you just delete this line? Otherwise I think Visual Studio will autogenerate a removal.